viz/metrics-api/stat_summary.go

```diff
@@ -366,6 +368,52 @@ func (s *grpcServer) trafficSplitResourceQuery(ctx context.Context, req *pb.Stat
 	return resourceResult{res: &rsp, err: nil}
 }
 
+func (s *grpcServer) policyResourceQuery(ctx context.Context, req *pb.StatSummaryRequest) resourceResult {
+
+	rows := make([]*pb.StatTable_PodGroup_Row, 0)
+	requestMetrics := make(map[rKey]*pb.BasicStats)
+	tcpMetrics := make(map[rKey]*pb.TcpStats)
+
+	if !req.SkipStats {
+		var err error
+		requestMetrics, tcpMetrics, err = s.getPolicyMetrics(ctx, req, req.TimeWindow)
+		if err != nil {
+			return resourceResult{res: nil, err: err}
+		}
+	}
+
+	keys := make([]rKey, 0)
+	for k := range requestMetrics {
+		keys = append(keys, k)
+	}
+	for k := range tcpMetrics {
+		keys = append(keys, k)
+	}
```

Comment:
This is the part still pending that you mentioned, where k8s objects need to be filtered?

Reply:
Also, the lack of info about the k8s objects results in tables with no names, like:
```console
$ bin/go-run cli viz stat server -n emojivoto
NAME   MESHED   SUCCESS      RPS   LATENCY_P50   LATENCY_P95   LATENCY_P99   TCP_CONN
            -   100.00%   1.2rps           1ms           1ms           1ms          4
```

Reply:
Yep, That's the reason. As we are directly querying the prometheus metrics.